src/snowflake/snowpark/session.py

```diff
@@ -23,7 +23,6 @@
 
 import cloudpickle
 import pkg_resources
-
 from snowflake.connector import ProgrammingError, SnowflakeConnection
```

Comment:
is this added by linter?

Reply:
Reran pre-commit to ensure the formatting is consistent.